apps/web/src/state/profile/helpers.ts

```diff
@@ -45,7 +45,7 @@ export const getUsername = async (address: string): Promise<string> => {
 
 export const getProfile = async (address: string): Promise<GetProfileResponse> => {
   try {
-    const client = publicClient({ chainId: ChainId.BSC })
+    const client = getViemClients({ chainId: ChainId.BSC })
```

Comment:
Why not this to match other changes?
```suggestion
    const client = viemClients[ChainId.BSC]
```

Reply:
It actually should be getViemClients everything I just forgot some lines I guess